ajax/libs/workbox/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "workbox",
+  "version": "3.0.0",
+  "filename": "workbox-sw.js",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "This module makes it easy to get started with the Workbox service worker libraries.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/workbox"
+  },
+  "keywords": [
```

Comment:
How about to add those keywords too
![image](https://user-images.githubusercontent.com/11975561/38542627-00bc707e-3cab-11e8-800e-f7590a2b9f74.png)